packages/react-instantsearch-hooks-dom/src/ui/Highlight.tsx

```diff
@@ -73,7 +94,7 @@ export function Highlight({
             ))}
 
             {!isLastPart && (
-              <span className={`${baseClassName}-separator`}>{separator}</span>
+              <span className={cx(classNames.separator)}>{separator}</span>
```

Comment:
Why do we call `cx()` here?
```suggestion
              <span className={classNames.separator}>{separator}</span>
```

Reply:
This is because each key in classNames was a string, number etc, but on reflection, that doesn't actually make sense, will change it to string